### PR TITLE
[CAS-1170] Add support for French

### PR DESCRIPTION
**Pull request by adasiewiczr**
https://stream-io.atlassian.net/browse/CAS-1170

### 🎯 Goal

Add French translations

### 🛠 Implementation details

Added French translations based on Flutter SDK and Google Translate.
You can find English strings here:
- [Client](https://github.com/GetStream/stream-chat-android/blob/develop/stream-chat-android-client/src/main/res/values/strings.xml)
- [Common](https://github.com/GetStream/stream-chat-android/blob/develop/stream-chat-android-ui-common/src/main/res/values/strings.xml)
- [UI Components](https://github.com/GetStream/stream-chat-android/tree/develop/stream-chat-android-ui-components/src/main/res/values)

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS (task created))
- [x] Reviewers added

### CODE REVIEW

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_common.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="stream_ui_mention">\@%s</string>
+    <string name="stream_ui_yesterday">Hier</string>
+
+    <!-- User Status (Used in Channel Actions Dialog and Message List Header) -->
+    <string name="stream_ui_user_status_online">En ligne</string>
+    <string name="stream_ui_user_status_last_seen">Vu pour la dernière fois %s</string>
+    <string name="stream_ui_user_status_last_seen_just_now">Last seen just now</string>
```
Comment (adasiewiczr) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_common.xml:9]: I have no idea how to translate this one

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_message_list.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    </plurals>
+
+    <!-- Message Actions Dialog -->
+    <string name="stream_ui_message_list_reply">Réponse</string>
+    <string name="stream_ui_message_list_thread_reply">Réponse au fil de discussion</string>
+    <string name="stream_ui_message_list_resend_message">Renvoyer</string>
+    <string name="stream_ui_message_list_copy_message">Copier le message</string>
+    <string name="stream_ui_message_list_edit_message">Modifier le message</string>
+    <string name="stream_ui_message_list_flag_message">Signaler un message</string>
+    <string name="stream_ui_message_list_mute_user">Utilisateur muter</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_message_list.xml:33]:
```suggestion
    <string name="stream_ui_message_list_mute_user">Mettre en sourdine l\'utilisateur</string>
```

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_message_list.xml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    <string name="stream_ui_message_list_unmute_user">Utilisateur unmute</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_message_list.xml:34]:
```suggestion
    <string name="stream_ui_message_list_unmute_user">Désactiver la mise en sourdine de l\'utilisateur</string>
```

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_message_list.xml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    <string name="stream_ui_message_list_block_user">Bloquer un utilisateur</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_message_list.xml:35]:
```suggestion
    <string name="stream_ui_message_list_block_user">Bloquer l\'utilisateur</string>
```

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_message_list.xml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    <string name="stream_ui_message_list_delete_message">Supprimer le message</string>
+    <plurals name="stream_ui_message_list_message_reactions">
+        <item quantity="one">%d Réaction aux messages</item>
+        <item quantity="other">%d Réactions aux messages</item>
+    </plurals>
+
+    <!-- Message Action Confirmations -->
+    <string name="stream_ui_message_list_delete_confirmation_title">Supprimer le message</string>
+    <string name="stream_ui_message_list_delete_confirmation_message">Voulez-vous vraiment supprimer définitivement ce message ?</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_message_list.xml:44]:
```suggestion
    <string name="stream_ui_message_list_delete_confirmation_message">Êtes-vous sûr de vouloir supprimer définitivement ce message ?</string>
```

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_message_list.xml [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    <string name="stream_ui_message_list_delete_confirmation_negative_button">Annuler</string>
+    <string name="stream_ui_message_list_flag_confirmation_title">Signaler un message</string>
+    <string name="stream_ui_message_list_flag_confirmation_message">Voulez-vous envoyer une copie de ce message à un nmodérateur pour une enquête plus approfondie?</string>
+    <string name="stream_ui_message_list_flag_confirmation_positive_button">Signaler</string>
+    <string name="stream_ui_message_list_flag_confirmation_negative_button">Annuler</string>
+
+    <!-- Attachment Viewer -->
+    <string name="stream_ui_message_list_attachment_load_failed">Le chargement a échoué en raison d\'une erreur inconnue.</string>
+    <string name="stream_ui_message_list_attachment_invalid_url">Le chargement a échoué en raison d\'une URL non valide.</string>
+    <string name="stream_ui_message_list_attachment_invalid_mime_type">Quelque chose s\'est mal passé. Impossible d\'ouvrir la pièce jointe: %s</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_message_list.xml:55]:
```suggestion
    <string name="stream_ui_message_list_attachment_invalid_mime_type">Un problème est survenu. Impossible d\'ouvrir la pièce jointe: %s</string>
```

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_message_list.xml [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+    <string name="stream_ui_message_list_attachment_display_error">Erreur. Le fichier ne peut pas être affiché</string>
+
+    <!-- Errors -->
+    <string name="stream_ui_message_list_error_mute_user">Échec à l\'utilisateur muet</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_message_list.xml:59]:
```suggestion
    <string name="stream_ui_message_list_error_mute_user">Échec de la mise en sourdine de l\'utilisateur</string>
```

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml [hunk 8]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="stream_ui_channel_list_empty">Aucune canaux disponible</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml:3]:
```suggestion
    <string name="stream_ui_channel_list_empty">Aucun canal disponible</string>
```

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+    <string name="stream_ui_channel_list_you">Toi</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml:4]:
```suggestion
    <string name="stream_ui_channel_list_you">Vous</string>
```

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+    <string name="stream_ui_channel_list_untitled_channel">Le canal sans nom</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml:5]:
```suggestion
    <string name="stream_ui_channel_list_untitled_channel">Canal sans nom</string>
```

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+
+    <!-- Channel Actions Dialog -->
+    <string name="stream_ui_channel_list_view_info">Voir les informations</string>
+    <string name="stream_ui_channel_list_leave_channel">Quitter le Group</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml:9]:
```suggestion
    <string name="stream_ui_channel_list_leave_channel">Quitter le Groupe</string>
```

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+        <item quantity="other">%1d Membres, %2d En ligne</item>
+    </plurals>
+
+    <string name="stream_ui_channel_list_delete_confirmation_title">Supprimer la canal</string>
+    <string name="stream_ui_channel_list_delete_confirmation_message">Êtes-vous sûr de vouloir supprimer définitivement ce\nmessage?</string>
+    <string name="stream_ui_channel_list_delete_confirmation_positive_button">Supprimer</string>
+    <string name="stream_ui_channel_list_delete_confirmation_negative_button">Annuler</string>
+
+    <!-- Errors -->
+    <string name="stream_ui_channel_list_error_hide_channel">Échec de la masquer la canal</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml:23]:
```suggestion
    <string name="stream_ui_channel_list_error_hide_channel">Echec du masquage du canal</string>
```

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+    <string name="stream_ui_channel_list_error_delete_channel">Échec de la suppression de la canal</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml:24]:
```suggestion
    <string name="stream_ui_channel_list_error_delete_channel">Échec de la suppression du canal</string>
```

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml [hunk 14]
```diff
@@ -0,0 +1,26 @@
...
+    <string name="stream_ui_channel_list_error_leave_channel">Échec de la sortie de la canal</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml:25]:
```suggestion
    <string name="stream_ui_channel_list_error_leave_channel">Échec de sortie du canal</string>
```

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml [hunk 15]
```diff
@@ -0,0 +1,26 @@
...
+    <string name="stream_ui_channel_list_leave_channel">Quitter le Group</string>
+    <string name="stream_ui_channel_list_dismiss_dialog">Annuler</string>
+    <string name="stream_ui_channel_list_delete_channel">Supprimer la conversation</string>
+    <plurals name="stream_ui_channel_list_member_info">
+        <item quantity="one">%1d Membre, %2d En ligne</item>
+        <item quantity="other">%1d Membres, %2d En ligne</item>
+    </plurals>
+
+    <string name="stream_ui_channel_list_delete_confirmation_title">Supprimer la canal</string>
+    <string name="stream_ui_channel_list_delete_confirmation_message">Êtes-vous sûr de vouloir supprimer définitivement ce\nmessage?</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_channel_list.xml:18]:
```suggestion
    <string name="stream_ui_channel_list_delete_confirmation_message">Êtes-vous sûr de vouloir supprimer définitivement cette conversation?</string>
```

File: stream-chat-android-ui-common/src/main/res/values-fr/strings.xml [hunk 16]
```diff
@@ -0,0 +1,22 @@
...
+
+    <!-- Media capture -->
+    <string name="stream_ui_message_input_capture_media">Capture d\'image ou vidéo</string>
+
+    <!-- Permissions -->
+    <string name="stream_ui_message_input_permission_camera_title">Caméra et stockage Autorisation d\'accès</string>
+    <string name="stream_ui_message_input_permission_camera_message">Des autorisations d\'accès à l\'appareil photo et au stockage sont nécessaires pour pouvoir prendre des photos et des vidéos</string>
+    <string name="stream_ui_message_input_permission_storage_title">Autorisations d\'accès au stockage</string>
+    <string name="stream_ui_message_input_permission_storage_message">Des autorisations d\'accès au stockage sont nécessaires pour pouvoir envoyer et enregistrer des photos, des vidéos, de la musique et les autres fichiers</string>
+    <string name="stream_ui_message_input_permission_setting_message">Activer les autorisations sur Paramètres de l\'application</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-common/src/main/res/values-fr/strings.xml:14]:
```suggestion
    <string name="stream_ui_message_input_permission_setting_message">Activez les autorisations dans les paramètres de l\'application</string>
```

File: stream-chat-android-ui-common/src/main/res/values-fr/strings.xml [hunk 17]
```diff
@@ -0,0 +1,22 @@
...
+    <string name="stream_ui_yesterday">Hier</string>
+
+    <!-- Media capture -->
+    <string name="stream_ui_message_input_capture_media">Capture d\'image ou vidéo</string>
+
+    <!-- Permissions -->
+    <string name="stream_ui_message_input_permission_camera_title">Caméra et stockage Autorisation d\'accès</string>
+    <string name="stream_ui_message_input_permission_camera_message">Des autorisations d\'accès à l\'appareil photo et au stockage sont nécessaires pour pouvoir prendre des photos et des vidéos</string>
+    <string name="stream_ui_message_input_permission_storage_title">Autorisations d\'accès au stockage</string>
+    <string name="stream_ui_message_input_permission_storage_message">Des autorisations d\'accès au stockage sont nécessaires pour pouvoir envoyer et enregistrer des photos, des vidéos, de la musique et les autres fichiers</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-common/src/main/res/values-fr/strings.xml:13]:
```suggestion
    <string name="stream_ui_message_input_permission_storage_message">Des autorisations d\'accès au stockage sont nécessaires pour pouvoir envoyer et sauvegarder des photos, des vidéos, de la musique et d\'autres fichiers.</string>
```

File: stream-chat-android-ui-common/src/main/res/values-fr/strings.xml [hunk 18]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!--Message Date-->
+    <string name="stream_ui_yesterday">Hier</string>
+
+    <!-- Media capture -->
+    <string name="stream_ui_message_input_capture_media">Capture d\'image ou vidéo</string>
+
+    <!-- Permissions -->
+    <string name="stream_ui_message_input_permission_camera_title">Caméra et stockage Autorisation d\'accès</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-common/src/main/res/values-fr/strings.xml:10]:
```suggestion
    <string name="stream_ui_message_input_permission_camera_title">Permission d\'accès aux caméras et au stockage</string>
```

File: stream-chat-android-ui-common/src/main/res/values-fr/strings.xml [hunk 19]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!--Message Date-->
+    <string name="stream_ui_yesterday">Hier</string>
+
+    <!-- Media capture -->
+    <string name="stream_ui_message_input_capture_media">Capture d\'image ou vidéo</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-common/src/main/res/values-fr/strings.xml:7]:
```suggestion
    <string name="stream_ui_message_input_capture_media">Capture d\'image ou de vidéo</string>
```

File: stream-chat-android-client/src/main/res/values-fr/strings.xml [hunk 20]
```diff
@@ -0,0 +1,15 @@
+<resources>
+
+    <!--Notification-->
+    <string name="stream_chat_notification_title">Message de discussion</string>
+    <string name="stream_chat_notification_content">Vous avez reçu un nouveau message</string>
+    <string name="stream_chat_notification_reply">Réponse</string>
```
Comment (sachaarbonel) [inline stream-chat-android-client/src/main/res/values-fr/strings.xml:6]:
```suggestion
    <string name="stream_chat_notification_reply">Répondre</string>
```

File: stream-chat-android-client/src/main/res/values-fr/strings.xml [hunk 21]
```diff
@@ -0,0 +1,15 @@
...
+    <string name="stream_chat_notification_type_hint">Tapez le message</string>
```
Comment (sachaarbonel) [inline stream-chat-android-client/src/main/res/values-fr/strings.xml:7]:
```suggestion
    <string name="stream_chat_notification_type_hint">Tapez votre message</string>
```

File: stream-chat-android-ui-common/src/main/res/values-fr/strings.xml (hunk 19 above)
Comment (sachaarbonel) [reply stream-chat-android-ui-common/src/main/res/values-fr/strings.xml:7]:
```suggestion
    <string name="stream_ui_message_input_capture_media">Capture d\'image ou de vidéo</string>
```

File: stream-chat-android-ui-common/src/main/res/values-fr/strings.xml [hunk 22]
```diff
@@ -0,0 +1,22 @@
...
+    <string name="stream_ui_message_input_permission_storage_title">Autorisations d\'accès au stockage</string>
+    <string name="stream_ui_message_input_permission_storage_message">Des autorisations d\'accès au stockage sont nécessaires pour pouvoir envoyer et enregistrer des photos, des vidéos, de la musique et les autres fichiers</string>
+    <string name="stream_ui_message_input_permission_setting_message">Activer les autorisations sur Paramètres de l\'application</string>
+    <string name="stream_ui_message_input_permissions_setting_button">Paramètres</string>
+
+    <!--Attachment View-->
+    <string name="stream_ui_message_list_attachment_load_failed">Le chargement a échoué en raison d\'une erreur inconnue.</string>
+    <string name="stream_ui_message_list_attachment_invalid_url">Le chargement a échoué en raison d\'une URL non valide.</string>
+    <string name="stream_ui_message_list_attachment_invalid_mime_type">Quelque chose s\'est mal passé. Impossible d\'ouvrir la pièce jointe: %s</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-common/src/main/res/values-fr/strings.xml:20]:
```suggestion
    <string name="stream_ui_message_list_attachment_invalid_mime_type">Un problème est survenu. Impossible d\'ouvrir la pièce jointe: %s</string>
```

File: stream-chat-android-ui-components/src/main/res/values-fr/strings_message_list.xml [hunk 23]
```diff
@@ -0,0 +1,63 @@
...
+    <!-- Message List Item -->
+    <string name="stream_ui_message_list_giphy_title">Giphy</string>
+    <string name="stream_ui_message_list_giphy_send">Envoyer</string>
+    <string name="stream_ui_message_list_giphy_cancel">Annuler</string>
+    <string name="stream_ui_message_list_giphy_shuffle">Mélanger</string>
+    <string name="stream_ui_message_list_ephemeral_message">Seulement visible par vous</string>
+    <string name="stream_ui_message_list_deleted_message">Message supprimé</string>
+    <string name="stream_ui_message_list_attachment_more_count">+%d</string>
+    <string name="stream_ui_message_list_attachment_uploading">Téléchargement %d / %d …</string>
+    <string name="stream_ui_message_list_attachment_upload_complete">Téléchargement terminé, traitement…</string>
```
Comment (sachaarbonel) [inline stream-chat-android-ui-components/src/main/res/values-fr/strings_message_list.xml:15]:
```suggestion
    <string name="stream_ui_message_list_attachment_upload_complete">Transfert terminé, traitement…</string>
```

**Review by sachaarbonel — CHANGES_REQUESTED** [review]
Added some reviews

**Comment by JcMinarro** [comment]
Thanks @sachaarbonel for the review

**Review by JcMinarro — APPROVED** [review]

**Pull request merged**